docker-compose.dev.yml

```diff
@@ -10,6 +10,8 @@ version: '3.4'
 services:
   db-postgres:
     container_name: seed_postgres
+    environment:
+      - PGDATA=/var/lib/postgresql/data/pgdata
```

Comment:
docker-compose.dev is intended to extend docker-compose.yml, so I don't think we have to redeclare this
```suggestion
```